app/src/main/java/org/novinger/jason/demoreader/ListActivity.java

```diff
@@ -1,47 +1,70 @@
 package org.novinger.jason.demoreader;
 
+import android.os.StrictMode;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.util.Log;
 import android.view.Menu;
 import android.view.MenuItem;
 import android.view.View;
+import android.view.ViewPropertyAnimator;
 import android.widget.AdapterView;
 import android.widget.ArrayAdapter;
 import android.widget.ListView;
 
+import com.google.gson.Gson;
+
+import org.novinger.jason.demoreader.datamodels.Article;
+
+import java.io.IOException;
 import java.util.ArrayList;
-import java.util.Collections;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
 
 public class ListActivity extends AppCompatActivity {
 
+    private final static int qty = 50;
+    private final static String feedUrl = "http://www.dailydot.com/api/v1/content/article/";
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_list);
 
+        StrictMode.ThreadPolicy policy = new StrictMode.ThreadPolicy.Builder().permitAll().build();
```

Comment:
generally you check network state explicitly...  http://developer.android.com/training/monitoring-device-state/connectivity-monitoring.html

the StrictMode stuff will just alert you so i don't think is useful for production